tests/test_snmpserver.rb

```diff
@@ -46,113 +46,55 @@ def setup
-    timeout = line.to_s.split(' ').last.to_i
-    assert_equal(timeout, snmpserver.aaa_user_cache_timeout)
+    # negative tests
+    assert_raises(Cisco::CliError) { s.aaa_user_cache_timeout = 86_401 }
+    assert_raises(Cisco::CliError) { s.aaa_user_cache_timeout = 0 }
   end
 
-  def test_snmpserver_sys_contact_get
-    snmpserver = SnmpServer.new
-    snmpserver.contact = 'test-contact'
-    if platform == :nexus
-      line = assert_show_match(command: 'show snmp | no-more',
-                               pattern: /sys contact: [^\n\r]+/)
-      contact = ''
-      contact = line.to_s.gsub('sys contact:', '').strip unless line.nil?
-    else
-      line = assert_show_match(command: 'show running-config snmp-server',
-                               pattern: /snmp-server contact [^\n\r]+/)
-      contact = ''
-      contact = line.to_s.gsub('snmp-server contact', '').strip unless line.nil?
-    end
-    # puts "contact : #{line}, #{snmpserver.contact}"
-    assert_equal(contact, snmpserver.contact)
-    # set to default
-    snmpserver.contact = snmpserver.default_contact
-  end
+  def test_contact
+    s = SnmpServer.new
+    # initial
+    default = s.default_contact
+    assert_equal(default, s.contact)
 
-  def test_snmpserver_sys_contact_set_invalid
-    snmpserver = SnmpServer.new
-    assert_raises(TypeError) do
-      snmpserver.contact = 123
-    end
-  end
+    # non-default
+    s.contact = 'minitest'
+    assert_equal('minitest', s.contact)
 
-  def test_snmpserver_sys_contact_set_zero_length
-    snmpserver = SnmpServer.new
-    newcontact = ''
-    snmpserver.contact = newcontact
-    assert_equal(newcontact, snmpserver.contact)
-    # set to default
-    snmpserver.contact = snmpserver.default_contact
-  end
+    # default
+    s.contact = default
+    assert_equal(default, s.contact)
 
-  def test_snmpserver_sys_contact_set_valid
-    snmpserver = SnmpServer.new
-    newcontact = 'mvenkata_test# contact'
-    snmpserver.contact = newcontact
-    if platform == :nexus
-      line = assert_show_match(command: 'show snmp | no-more',
-                               pattern: /sys contact: [^\n\r]+/)
-      # puts "line: #{line}"
-      contact = line.to_s.gsub('sys contact:', '').strip
-    else
-      line = assert_show_match(command: 'show running-config snmp-server',
-                               pattern: /snmp-server contact [^\n\r]+/)
-      contact = ''
-      contact = line.to_s.gsub('snmp-server contact', '').strip unless line.nil?
-    end
-    assert_equal(contact, newcontact)
-    # set to default
-    snmpserver.contact = snmpserver.default_contact
+    # empty
+    s.contact = ''
+    assert_equal('', s.contact)
+
+    # negative test
+    assert_raises(TypeError) { s.contact = 123 }
   end
 
-  def test_snmpserver_sys_contact_set_special_chars
+  def test_contact_special_chars
     snmpserver = SnmpServer.new
     # newcontact = "Test{}(%tuvy@_cisco contact$#!@1234^&*()_+"
```

Comment:
Sure. I assumed it was just another test input somebody wanted to keep around. I'll remove it.